kubernetes/launch-job.sh

```diff
@@ -20,28 +21,35 @@ usage() {
   echo "  -m GPUS, --victimplay-max-gpus GPUS"
   echo "    Maximum number of GPUs to use for victimplay."
   echo "    default: twice the minimum number of GPUs."
+  echo "  -r, --resume TIMESTAMP"
+  echo "    Resume a previous run. If this flag is given, the PREFIX argument"
+  echo "    must exactly be match the run to be resumed, and the TIMESTAMP"
+  echo "    argument should match the timestamp attached to the name of the"
+  echo "    previous run's output directory. The use of the --use-weka flag"
+  echo "    must also exactly match that of the previous run."
   echo "  -w, --use-weka"
   echo "    Store results on the go-attack Weka volume instead of the CHAI NAS"
   echo "    volume."
   echo
   echo "Optional arguments should be specified before positional arguments."
 }
 
-NUM_POSITIONAL_ARGUMENTS=1
-
 MIN_VICTIMPLAY_GPUS=${DEFAULT_NUM_VICTIMPLAY_GPUS}
 # Command line flag parsing (https://stackoverflow.com/a/33826763/4865149)
-while [ "$#" -gt ${NUM_POSITIONAL_ARGUMENTS} ]; do
+while true; do
```

Comment:
`"$#" -gt ${NUM_POSITIONAL_ARGUMENTS}` check didn't work because then `./launch-job -h` would be interpreted as launching a job with `PREFIX=-h` rather than printing a help message